src/simplicial-hott/03-extension-types.rzk.md

```diff
@@ -130,8 +130,8 @@ This equivalence is functorial in the following sense:
   ( Y : ψ → X → U)
   ( f : (t : ϕ) → (x : X) → Y t x)
   : Equiv
-      ( (t : ψ) → ((x : X) → Y t x) [ϕ t ↦ f t])
-      ( (x : X) → (t : ψ) → Y t x [ϕ t ↦ f t x])
+      ( ( t : ψ) → ((x : X) → Y t x) [ϕ t ↦ f t])
+      ( ( x : X) → (t : ψ) → Y t x [ϕ t ↦ f t x])
```

Comment:
These spaces also look unnecessary and weird to me, but again it's a matter of preference.